KubeArmor/build/kubearmor-test-crio.yaml

```diff
@@ -57,6 +57,7 @@ spec:
         - containerPort: 32767
         securityContext:
           privileged: true
+          readOnlyRootFilesystem: true
         terminationMessagePath: /dev/termination-log
```

Comment:
I am not sure what is the termination log for, but would readonlyfilesystem impact that?

Reply:
I checked, it does not impact it.

you can check in the  pod by :
`$ echo "random_string" >> /dev/termination-log`
`$ cat/dev/termination-log`  should output `random_string`